Description: Fix FTBFS

Author: Sudip Mukherjee <sudipm.mukherjee@gmail.com>
Bug-Debian: https://bugs.debian.org/975144
Forwarded: no

---

--- qpdfview-0.4.18.orig/sources/model.h
+++ qpdfview-0.4.18/sources/model.h
@@ -27,6 +27,7 @@ along with qpdfview.  If not, see <http:
 #include <QtPlugin>
 #include <QWidget>
 #include <QVector>
+#include <QPainterPath>
 
 class QAbstractItemModel;
 class QColor;
